extensions/Worker.Extensions.Kafka/src/KafkaTriggerAttribute.cs

```diff
@@ -77,25 +77,25 @@ public KafkaTriggerAttribute(string brokerList, string topic)
         /// Default: ""
         /// ssl.key.location in librdkafka
         /// </summary>
-        public string? SslKeyLocation { get; set; }
+        public string SslKeyLocation { get; set; }
 
         /// <summary>
         /// Path to CA certificate file for verifying the broker's certificate.
         /// ssl.ca.location in librdkafka
         /// </summary>
-        public string? SslCaLocation { get; set; }
+        public string SslCaLocation { get; set; }
 
         /// <summary>
         /// Path to client's certificate.
         /// ssl.certificate.location in librdkafka
         /// </summary>
-        public string? SslCertificateLocation { get; set; }
+        public string SslCertificateLocation { get; set; }
 
         /// <summary>
         /// Password for client's certificate.
         /// ssl.key.password in librdkafka
```

Comment:
Q: Is this a valid comment to present here?

Reply:
Comment in line with the extension.  https://github.com/Azure/azure-functions-kafka-extension/blob/82de33298da77b7e5066db8d6b902370641ae765/src/Microsoft.Azure.WebJobs.Extensions.Kafka/Output/KafkaAttribute.cs#L132